README.md

```diff
@@ -2,11 +2,11 @@
 
 **Maintainer:** Andreas Fackler (andreas.fackler@maidsafe.net)
 
-|Crate|Linux/OS X|Windows|Coverage|Issues|
-|:---:|:--------:|:-----:|:------:|:----:|
-|[![](http://meritbadge.herokuapp.com/accumulator)](https://crates.io/crates/accumulator)|[![Build Status](https://travis-ci.org/maidsafe/accumulator.svg?branch=master)](https://travis-ci.org/maidsafe/accumulator)|[![Build status](https://ci.appveyor.com/api/projects/status/1imtexgsshnpxnvn/branch/master?svg=true)](https://ci.appveyor.com/project/MaidSafe-QA/accumulator/branch/master)|[![Coverage Status](https://coveralls.io/repos/maidsafe/accumulator/badge.svg)](https://coveralls.io/r/maidsafe/accumulator)|[![Stories in Ready](https://badge.waffle.io/maidsafe/accumulator.png?label=ready&title=Ready)](https://waffle.io/maidsafe/accumulator)|
+|Crate|Linux/OS X|Windows|Issues|
+|:---:|:--------:|:-----:|:----:|
+|[![](http://meritbadge.herokuapp.com/accumulator)](https://crates.io/crates/accumulator)|[![Build Status](https://travis-ci.org/maidsafe/accumulator.svg?branch=master)](https://travis-ci.org/maidsafe/accumulator)|[![Build status](https://ci.appveyor.com/api/projects/status/1imtexgsshnpxnvn/branch/master?svg=true)](https://ci.appveyor.com/project/MaidSafe-QA/accumulator/branch/master)|[![Stories in Ready](https://badge.waffle.io/maidsafe/accumulator.png?label=ready&title=Ready)](https://waffle.io/maidsafe/accumulator)|
 
-| [API Docs - master branch](http://docs.maidsafe.net/accumulator/master) | [MaidSafe website](https://maidsafe.net) | [SAFE Dev Forum](https://forum.safedev.org) | [SAFE Network Forum](https://safenetforum.org) |
+| [API Docs](http://docs.rs/accumulator/) | [MaidSafe website](https://maidsafe.net) | [SAFE Dev Forum](https://forum.safedev.org) | [SAFE Network Forum](https://safenetforum.org) |
```

Comment:
better to move it after `crates` column, call it `Documentation` and provide a badge image like here: https://github.com/maidsafe/safe_client_libs/blob/28978cd10aad433332950284a49e96d881242ef9/safe_core/README.md